lite/examples/sound_classification/android_compose/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
There're two .gitignore files. Please unify them into a single file.

Reply:
It's a normal setup for Android apps to have two `.gitignore` files

Reply:
+1 to @yrezgui. Thanks!